types/google-apps-script/google-apps-script.card-service.d.ts

```diff
@@ -290,6 +290,10 @@ declare namespace GoogleAppsScript {
       newUniversalActionResponseBuilder(): UniversalActionResponseBuilder;
       newUpdateDraftActionResponseBuilder(): UpdateDraftActionResponseBuilder;
       newUpdateDraftBodyAction(): UpdateDraftBodyAction;
+      newUpdateDraftBccRecipientsAction(): UpdateDraftBccRecipientsAction;
```

Comment:
```suggestion
      newUpdateDraftBccRecipientsAction(): UpdateDraftBccRecipientsAction;
      newUpdateDraftBodyAction(): UpdateDraftBodyAction;
```
Sort alpabetical order